c2corg_ui/static/js/documentediting.js

```diff
@@ -351,10 +351,16 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
     if (this.modelName_ === 'outing') {
       // adapt the Object for what's expected at the API side + format the outing
       this.formatOuting_(data);
+      var userIds = [this.auth_.userData.id];
```

Comment:
With https://github.com/c2corg/v6_api/pull/293 being merged, outings can now be created like this:

https://github.com/c2corg/v6_api/blob/9485e78/c2corg_api/tests/views/test_outing.py#L344-L367